test/resources/invalid/typecheck/InvalidStringArray.flo

```diff
@@ -0,0 +1,6 @@
+action STUB_ACTION = stub();
+
+flow MY_FLOW = {
+    array myStringArray = [3, 1, 67];
```

Comment:
I think we need a new syntax for arrays, because there's nothing here that indicates this should be invalid.

Perhaps consider typescript's: `string[]`, then we can type check that `number[] != string[]` in this test case.

Reply:
I completely agree, let's do that

Reply:
Changed the tests to reflect this grammar change